README.rst

```diff
@@ -52,7 +52,7 @@ The MinIO web UI can be accessed at http://<MINIO_HOST>. The credentials for acc
 Troubleshooting
 ---------------
 
-This Tutor plugin is maintained by Régis Behmo from `Overhang.IO <https://overhang.io>`__. Community support is available from the official `Open edX forum <https://discuss.openedx.org>`__. Do you need help with this plugin? See the `troubleshooting <https://docs.tutor.overhang.io/troubleshooting.html>`__ section from the Tutor documentation.
+This Tutor plugin is maintained by Fahad Khalid from `Arbisoft.com <https://arbisoft.com/>`__. Community support is available from the official `Open edX forum <https://discuss.openedx.org>`__. Do you need help with this plugin? See the `troubleshooting <https://docs.tutor.overhang.io/troubleshooting.html>`__ section from the Tutor documentation.
```

Comment:
I suggest we link to Edly instead:

    `Edly <https://edly.io>`__

Reply:
Done